tests/wake-format/basic/basic.wake

```diff
@@ -895,6 +895,43 @@ target writeImp inputs mode path content =
     | getJobOutput
 
 export def makeJSONRunner (plan: JSONRunnerPlan): Runner =
+  def json =
+    # comment
+    ( 
+      JObject 
+      "label"       :-> JString label,
+      "command"     :-> command     | map JString | JArray,
+      "environment" :-> environment | map JString | JArray
+    )
+    (
```

Comment:
did you mean for a comma to be here or something? In fact none of this is particularly valid wake code lol. I'm not sure what you were trying to do.

Reply:
no comma, the goal was two have something multiline on the LHS of the apply.

so the main test is 

`Constructor ( some multiline RHS)`

while this bonus test is

`(some multiline constructor) (some multiline RHS)`